external/warc/src/main/java/com/digitalpebble/stormcrawler/warc/WARCSpout.java

```diff
@@ -59,6 +59,8 @@
 import com.digitalpebble.stormcrawler.spout.FileSpout;
 import com.digitalpebble.stormcrawler.util.ConfUtils;
 
+import parquet.it.unimi.dsi.fastutil.Arrays;
```

Comment:
I presume this is already in our dependencies but I'd rather just have a static final somewhere in our code

Reply:
Is this a shaded class? It's `it.unimi.dsi.fastutil.Arrays` in my code. If it's shaded, you don't want to be using it in your code.

Reply:
Ok, I'll change it. Also to be used for #854.

Reply:
Ok, it's now defined in the class Constants:
```java
/** Maximum array size, safe value on any JVM */
public static final int MAX_ARRAY_SIZE = Integer.MAX_VALUE - 8;
```